test/javascripts/initializers_test.coffee

```diff
@@ -106,10 +112,11 @@ describe 'Initializers', ->
         httpUrl: "somewhere"
         fullRefresh: false
         refreshOnSuccess: "foo"
+        refreshOnSuccessExcept: null
         refreshOnError: null
         refreshOnErrorExcept: null
 
-    it 'passes through null for missing full-refresh-on-error-except', ->
+    it 'respects full-refresh-on-error-except', ->
```

Comment:
`full-refresh-on-error-except` is a bit of a misnomer as it can also be a partial refresh on error except:
```
    if @refreshOnError || @refreshOnErrorExcept
      Page.refresh
        response: xhr
        onlyKeys: @refreshOnError
        exceptKeys: @refreshOnErrorExcept
```